packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -58,6 +58,42 @@ jobs:
 "
 `;
 
+exports[`lib should generate bitbucket pipelines config 1`] = `
+"image: node:16.18
+
+clone:
+  depth: full
+
+definitions:
+  steps:
+    - step: &agent
```

Comment:
Is the & intentional here?

Reply:
Yeah, that's the syntax BitBucket uses to store a reference to a reusable step